src/main/java/com/github/imas/rdflint/InteractiveMode.java

```diff
@@ -258,21 +257,23 @@ public void complete(LineReader reader, ParsedLine line, List<Candidate> candida
       }
 
       // prefix completer
+      Map<String, String> prefixMap = this.model.getNsPrefixMap();
       int idxBefore1 = line.words().size() - 2;
       int idxBefore2 = line.words().size() - 3;
       if (idxBefore1 >= 0 && "PREFIX".equals(line.words().get(idxBefore1).toUpperCase())) {
-        Stream.of(PREFIX_MAP)
-            .map(s -> s.split(" ")[0])
+        prefixMap.keySet().stream()
             .filter(s -> s.startsWith(line.word()))
```

Comment:
```suggestion
             .filter(s -> (s + ":").startsWith(line.word()))
```

プレフィックスを手入力し終えたあとでも、タブで次の補完に移動できるようにしました。

Reply:
ありがとうございます、対応します